src/traffic-collector/azext_traffic_collector/aaz/latest/network_function/traffic_collector/_update.py

```diff
@@ -0,0 +1,675 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "network-function traffic-collector update",
+)
+class Update(AAZCommand):
+    """Updates a Azure Traffic Collector resource
+
+    :example: Updates a azure traffic collector resource
+        az az network-function traffic-collector update --resource-group rg1 --azure-traffic-collector-name atc1 --tags key=value
+    """
+
+    _aaz_info = {
+        "version": "2022-08-01",
+        "resources": [
+            ["mgmt-plane", "/subscriptions/{}/resourcegroups/{}/providers/microsoft.networkfunction/azuretrafficcollectors/{}", "2022-08-01"],
+        ]
+    }
+
+    AZ_SUPPORT_NO_WAIT = True
+
+    AZ_SUPPORT_GENERIC_UPDATE = True
+
+    def _handler(self, command_args):
+        super()._handler(command_args)
+        return self.build_lro_poller(self._execute_operations, self._output)
+
+    _args_schema = None
+
+    @classmethod
+    def _build_arguments_schema(cls, *args, **kwargs):
+        if cls._args_schema is not None:
+            return cls._args_schema
+        cls._args_schema = super()._build_arguments_schema(*args, **kwargs)
+
+        # define Arg Group ""
+
+        _args_schema = cls._args_schema
+        _args_schema.azure_traffic_collector_name = AAZStrArg(
+            options=["-n", "--name", "--azure-traffic-collector-name"],
+            help="Azure Traffic Collector name",
+            required=True,
+            id_part="name",
+        )
+        _args_schema.resource_group = AAZResourceGroupNameArg(
+            required=True,
+        )
+
+        # define Arg Group "Parameters"
+
+        _args_schema = cls._args_schema
+        _args_schema.location = AAZResourceLocationArg(
+            arg_group="Parameters",
+            help="Resource location.",
+            fmt=AAZResourceLocationArgFormat(
+                resource_group_arg="resource_group",
+            ),
+        )
```

Comment:
We've released a new version of [aaz-dev-tools](https://github.com/Azure/aaz-dev-tools/releases/tag/v0.10.2) which fixed the location argument appeared in update commands. You can update and reload swagger in workspace editor.

Reply:
When i run below command, it installs the new version of aaz-dev-tools which is 0.10.2. When I reload swagger and generate again, i dont see any difference. Is that ok?

`pip install $(Invoke-WebRequest https://api.github.com/repos/Azure/aaz-dev-tools/releases/latest -UseBasicParsing | % { $_.Content } | ConvertFrom-Json | % { $_.assets.browser_download_url } | Select-String -Pattern "https.*.whl" -AllMatches | % { $_.Matches } | % { $_.Value })`

Reply:
Did you add `x-ms-mutability` in swagger?
![image](https://user-images.githubusercontent.com/69238381/188051023-e9b61486-2eb4-49b5-b9b3-ed22340c1de9.png)

Reply:
If you add `x-ms-mutability` for location, the `update` command will not have `--location` argument.